docs/user/projectnodes.md

```diff
@@ -9,25 +9,27 @@ very quick and easy to securely send and receive messages between different
 instances in a team.
```

Comment:
We should add a line to say the nodes are only available on the Team/Enterprise plans

Reply:
Updated in [6437fb5](https://github.com/FlowFuse/flowfuse/pull/3074/commits/6437fb5f91422d7e1ff3edc8d42eb891b53b35cf)

Adds the line "The project nodes are only available in the Team and Enterprise tiers of FlowFuse."